FreeType.xs

```diff
@@ -628,7 +628,7 @@ qefft2_face_fixed_sizes (Font_FreeType_Face face)
         int i;
         FT_Bitmap_Size *size;
         HV *hash;
-        double pt, ppem;
+        double pt = 12.0, ppem;
```

Comment:
On 2nd thought, freetype **can** cope with zero, so zero it will do. I'll change that.